CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+# Contributing to libseccomp-rs
+
+We're really glad you're interested in contributing to libseccomp-rs!
+This document has a few pointers and guidelines to help you get started.
+It is not perfect, and there will always be exceptions to the rules described
+here, but by following the instructions below you should have a much easier time
+getting your work merged with the upstream project.
+
+To have a welcoming and inclusive project, libseccomp-rs uses the Rust project's
+[Code of Conduct][code-of-conduct]. All contributors are expected to follow it.
+
+[code-of-conduct]: https://www.rust-lang.org/conduct.html
+
```

Comment:
```suggestion
## Questions

You can use [GitHub Discussions] to ask questions (Category Q&A).
As for issues, you should first search if you question might be already asked (and answered).

[GitHub Discussions]: https://github.com/libseccomp-rs/libseccomp-rs/discussions
```

Reply:
I added the section. Thank you for the suggestions.